app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,78 @@
+class ProjectsController < ApplicationController
+	def index
+		@project = current_user.projects
+	end
+
+	def new
+		@project = Project.new
+	end
+
+	def create
+    @project = Project.new(project_params)
+
+    if @project.save
+      @project.users<<current_user
+      redirect_to projects_path,  notice: "Your project was saved"
+    else
+      render "new"
+    end
+  end
+
+  def edit
+    @project = Project.find(params[:id])
+  end
+
+  def update
+    @project = Project.find(params[:id])
+      if @project.update(project_params)
+        flash[:notice] = "You have updated your project successfully."
+        redirect_to projects_path
+      else
+        render :edit
+      end
+  end
+
+  def destroy
+    @project = Project.find(params[:id])
+    
+    if @project.destroy
+        redirect_to projects_url, notice: "Project deleted."
+    end
+  end
+
+  def show_email
+    @email = params[:email]
+  end
+
+  def invite  
```

Comment:
it's not the same, InvitationsController invites someone to your team and whole app. And by invite method in ProjectsController you can invite to particular project one of members of your team.